src/routes/eth.route.js

```diff
@@ -39,16 +39,16 @@ router.post('/balances', async (req, res) => {
   }
   let tokenAddressList
   if (paramData.tokenAddressList) {
-    tokenAddressList = paramData.tokenAddressList.split(separator)
+    tokenAddressList = JSON.parse(paramData.tokenAddressList)
```

Comment:
This will change the tokenAddressList from comma separated value to JSON string. HB client update will be needed for this change.

Reply:
yeah, that is already handled in the PR 2690 mentioned above.